pkg/apis/discovery/v1alpha1/clusterducktype_types_test.go

```diff
@@ -134,9 +134,77 @@ func TestAPIVersion(t *testing.T) {
 
 	for name, tc := range tests {
 		t.Run(name, func(t *testing.T) {
-			got := APIVersion(tc.in)
+			got := tc.in.GroupVersion()
 			if !cmp.Equal(got, tc.want) {
-				t.Errorf("APIVersion (-want, +got) = %v",
+				t.Errorf("GroupVersion (-want, +got) = %v",
+					cmp.Diff(tc.want, got))
+			}
+		})
+	}
+}
+
+func TestResourceMeta_Group(t *testing.T) {
+	tests := map[string]struct {
+		in   ResourceMeta
+		want string
+	}{
+		"empty": {
+			in:   ResourceMeta{},
+			want: "",
+		},
+		"version and group set": {
+			in: ResourceMeta{
+				APIVersion: "this.group/v1",
+			},
+			want: "this.group",
+		},
+		"only version": {
+			in: ResourceMeta{
+				APIVersion: "v1",
+			},
+			want: "",
+		},
+	}
+
+	for name, tc := range tests {
+		t.Run(name, func(t *testing.T) {
+			got := tc.in.Group()
+			if !cmp.Equal(got, tc.want) {
```

Comment:
Same here, can simply compare the output of Diff to "" and use that in the error.